.github/workflows/test.yaml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+on: ["push", "pull_request"]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+        cache: 'npm'
+    - run: npm install
```

Comment:
I'm definitely not an expert on Github actions, but is it ok to add the hyphen to this line? It seems like the `run: npm install` is inside the -` name: Setup Node`, isn't it?

Reply:
this is as it should be. We do it the same for `nucypher-contracts` https://github.com/nucypher/nucypher-contracts/blob/710b68713b5b8fece0d0b1c5de376dba1d7d68e7/.github/workflows/main.yaml#L35